### ISSUE TITLE
Add an option to restart PowerShell host per execution

### ISSUE THREAD

**Issue by adamdriscoll** [microsoft/poshtools#292]
Developing a C# assembly and load it up in a PowerShell script run by the Visual Studio PowerShell host. Currently, the PowerShell host continues to run and I can no long build my C# assembly because it's loaded into the host and locked.

**Comment by ASayre** [comment]
@adamdriscoll 
I presume the ISE/Powershell cmd prompt acts the same way, and require a restart to release the assembly?

**Comment by adamdriscoll** [comment]
Correct. This kind of goes along the same lines as cross-language debugging for the PSHost. Would be great to step right from a script into my assembly, compile and do it again.
